common/djangoapps/util/enterprise_helpers.py

```diff
@@ -0,0 +1,125 @@
+"""
+Helpers to access the enterprise app
+"""
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+try:
+    from enterprise.models import EnterpriseCustomer
+    from enterprise.tpa_pipeline import (
+        active_provider_requests_data_sharing,
+        active_provider_enforces_data_sharing,
+        get_enterprise_customer_for_request,
+    )
+
+except ImportError:
+    pass
```

Comment:
Is there anything that we should do here in addition to passing back control?  eg. write to a log file?